src/entry.js

```diff
@@ -0,0 +1,18 @@
+const fromPairs = require('lodash/fromPairs');
+const assign = require('lodash/assign');
+
+const mergeAll = (accum, current) => assign(accum, current);
+
+const contexts = [
+  require.context('./atoms', true, /.*\.js/),
+  require.context('./molecules', true, /.*\.js/),
+  require.context('./organisms', true, /.*\.js/),
+  require.context('./templates', true, /.*\.js/),
+  // require.context('./pages', true, /.*\.js/),
+];
+
+module.exports = contexts.map((context) => fromPairs(context.keys().map((component) => {
+  const componentName = component.match(/.*\/(.*)\/index.js/)[1];
+
+  return [componentName, context(component).default];
+}))).reduce(mergeAll, {});
```

Comment:
This is a little complicated, but what we're doing here is dynamically identifying every file that is `**/*/Component/index.js` in the atoms, molecules, organisms and templates directories and exporting them at the top level